packages/-build-infra/src/node-test-helpers/setup-test-environment.js

```diff
@@ -1,3 +1,4 @@
+/* global beforeEach afterEach */
```

Comment:
Shouldn't need to use `globals` here, points to an issue with eslint config

Reply:
Fixed